htdocs/install/mysql/tables/llx_paymentuser.sql

```diff
@@ -16,14 +17,14 @@
 --
 -- ===================================================================
 
-create table llx_payment_expensereport
+create table llx_paymentuser
```

Comment:
Why renaming this table. Current structure and usage seems ok.  ?

Reply:
Done

Reply:
@eldy